demo/src/app/components/datepicker/demos/calendars/datepicker-calendars.html

```diff
@@ -1,17 +1,7 @@
-<p>Islamic Civil calendar</p>
+<span style="float: left;">
```

Comment:
Can we utilize Bootstrap's [grid system](https://v4-alpha.getbootstrap.com/layout/grid/#equal-width) here for the two column layout instead of floats?